pytest_stoq/stoq.py

```diff
@@ -48,6 +49,8 @@ def _setup_test_environment(request):
     port = int(os.environ.get("STOQLIB_TEST_PORT") or 0)
     quick = request.config.getvalue("quick_mode")
     quick = quick or _to_falsy(os.environ.get("STOQLIB_TEST_QUICK", None))
+    extra_plugins = request.config.inicfg.get("STOQ_PLUGINS", None)
+    extra_plugins = extra_plugins.split(',') if extra_plugins else None
```

Comment:
it would be nice to have tests for this. there are already some tests on this function that you can use as a base [1] 

[1] https://github.com/stoq/pytest-stoq/blob/master/tests/test_stoq.py#L88